local/network.go

```diff
@@ -241,7 +242,8 @@ func newNetwork(
 ) (*localNetwork, error) {
 	var err error
 	if rootDir == "" {
-		rootDir, err = os.MkdirTemp("", fmt.Sprintf("%s-*", rootDirPrefix))
+		rootDir = path.Join(os.TempDir(), rootDirPrefix)
```

Comment:
We should use [filepath.Join](https://pkg.go.dev/path/filepath) instead of `path.Join`. 

From https://pkg.go.dev/path:

```
The path package should only be used for paths separated by forward slashes, such as the paths in URLs. This package does not deal with Windows paths with drive letters or backslashes; to manipulate operating system paths, use the path/filepath package.
```